src/features/encoders/oxiPNG/worker/oxipngEncode.ts

```diff
@@ -40,9 +40,12 @@ export default async function encode(
   options: EncodeOptions,
 ): Promise<ArrayBuffer> {
   if (!wasmReady) {
-    wasmReady = threads().then((hasThreads: boolean) => hasThreads ? initMT() : initST());
+    wasmReady = threads().then((hasThreads: boolean) =>
+      hasThreads ? initMT() : initST(),
+    );
```

Comment:
Hah, I guess prettier didn't pick this up last time

Reply:
Weirddd.